### ISSUE TITLE
how do i restore a model

### ISSUE THREAD

**Issue by snurrarsnabbt** [jiny2001/dcscn-super-resolution#44]
Hello, 
How do I do if I want to restore a model I already have trained, and continue training on that model?

**Comment by jiny2001** [comment]
Hi,
You can use --load_model_name="your_model_filename" option. That option will load the weights
from the file and then start training again with different learning rate, optimizer, loss function or whatever you want.
